modules/dists/CyclicDist.chpl

```diff
@@ -166,6 +165,56 @@ domain or array.
     var CyclicDom2 = Cyclic.createDomain(1..5, 1..5);
     var CyclicArr2 = Cyclic.createArray(1..5, 1..5, real);
 
+The helper methods on ``Cyclic`` have the following signatures:
+
+  .. function:: proc type Cyclic.createDomain(dom: domain, targetLocales = Locales)
+
+    Create a domain over a Cyclic Distribution.
+
+  .. function:: proc type Cyclic.createDomain(rng: range(?)..., targetLocales = Locales)
+
+    Create a domain over a Cyclic Distribution constructed from a series of
+    ranges.
+
+  .. function:: proc type Cyclic.createArray(dom: domain, type eltType, targetLocales = Locales)
+
+    Create a default initialized array over a Cyclic Distribution using the
+    given domain.
+
+  .. function:: proc type Cyclic.createArray(rng: range(?)..., type eltType, targetLocales = Locales)
+
+    Create a default initialized array over a Cyclic Distribution using a
+    domain constructed from the series of ranges.
+
+  .. function:: proc type Cyclic.createArray(dom: domain, type eltType, initExpr, targetLocales = Locales)
+
+    Create an array over a Cyclic Distribution using the given domain.
+
+    The array's values are initialized using ``initExpr`` which can be any of
+    the following:
+
+    * a value coercible to ``eltType`` — all elements of the array will have this value
+    * an iterator expression that yields values of type ``eltType`` — the iterator
+      will be assigned into the array
```

Comment:
"the iterator will be assigned into the array" --> "the array elements will be initialized with the values yielded by the iterator" or something similar